core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -937,4 +938,17 @@ default HttpResponse recover(Function<? super Throwable, ? extends HttpResponse>
         requireNonNull(function, "function");
         return of(new RecoverableStreamMessage<>(this, function, /* allowResuming */ false));
     }
+
+	default HttpResponse recover(Class<? extends Throwable> causeClass,
+								 Function<? super Throwable, ? extends HttpResponse> function) {
```

Comment:
Thanks for your contribution. 😄 
We use four spaces instead of tabs. Could you check the [developer guide](https://armeria.dev/community/developer-guide#setting-up-your-ide) and apply it for this PR, please?

Reply:
Oops... sorry i missed it 😅
i changed tab to four space and re order imports